Documentation/compatibility/workflow-designer-accessibility.md

```diff
@@ -0,0 +1,123 @@
+## Accessibility improvements in WF workflow designer
+
+### Scope
+Minor
+
+### Version Introduced
+4.7.1
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+The Windows Workflow Foundation (WF) workflow designer is improving how it works with accessibility technologies. These improvements include the following changes:
+- Tab order changed to left->right & top->bottom in some controls
+  - Initialize correlation window for setting correlation data for the InitializeCorrelation activity
+  - Content definition window for Receive, Send, SendReply, and ReceiveReply activities
```

Comment:
link to classes for each activity (e.g., `T:System.ServiceModel.Activities.Receive`)?